### ISSUE TITLE
Printing out the probabilities of the softmax function

### ISSUE THREAD

**Issue by jialongge** [satyenrajpal/Concrete-Crack-Detection#15]
Dear @satyenrajpal , I would like to print out the probabilities of each class(i believe its the output of the softmax layer), just before it gets sent into y_pred_cls and gets classified into one of the classes. Are there any ways I can do the printing? I tried printing directly but it prints me `%s Tensor("Softmax:0", shape=(?, 2), dtype=float32)`

**Comment by satyenrajpal** [comment]
Hi! Since this is tensorflow, you will have to print it in a session such as `sess.run(prob)` where `prob` is the probability vector. Please refer to the TensorFlow documentation for reference.

**Comment by jialongge** [comment]
Hi @satyenrajpal Thanks for the reply! I had a difficult time printing the probabilities. In my use case, I want to be able to print the probabilities while running Running.py input images. 

I believe the probabilities is in y_pred in the Train_CD.py file, which Running.py imports. The session ran is also in Running,py. I tried to print the y_pred inside a session in Train_CD.py(though I am running Running.py) but it didnt return me anything. I tried printing it from within Running.py, but it doesnt give me the desired probabilities as well.

I believe the problem is because in Running.py, the model is being imported, instead of being local. I am not sure if my understanding is correct. Sorry if I am wrong! I am a beginner =( I tried numerous times but cant seem to print correctly.
